nautobot/ipam/api/serializers.py

```diff
@@ -344,15 +233,7 @@ def to_representation(self, instance):
 
 class ServiceSerializer(NautobotModelSerializer, TaggedModelSerializerMixin):
     url = serializers.HyperlinkedIdentityField(view_name="ipam-api:service-detail")
-    device = NestedDeviceSerializer(required=False, allow_null=True)
-    virtual_machine = NestedVirtualMachineSerializer(required=False, allow_null=True)
     protocol = ChoiceField(choices=ServiceProtocolChoices, required=False)
```

Comment:
Is this needed or does it "just work"?

Reply:
Gonna open a backlog item on this.

Reply:
https://github.com/nautobot/nautobot/issues/3631